src/components/create-page/nft-properties.component.tsx

```diff
@@ -0,0 +1,225 @@
+    if (properties.length <= 10) setPropertiesModalMessage('');
+  }, [properties]);
+
+  const handlePropsModal = () => setPropsModalOpen((prev) => !prev);
+
+  const handleAddMoreProps = (key: string, value: string) => {
+    const itemsWithSameKey = properties.filter((prop: ItemProperty) => prop.key === key);
+
+    if (itemsWithSameKey.length > 0) {
+      setPropertiesModalMessage(propertiesModalMessages.NotRepeteadAllowed);
+    } else if (key === '' || value === '') {
+      setPropertiesModalMessage(propertiesModalMessages.ProvideData);
+    } else if (properties.length >= 10) {
+      setPropertiesModalMessage(propertiesModalMessages.MaxLength);
+    } else if (key.length > 9 || value.length > 9) {
+      setPropertiesModalMessage(propertiesModalMessages.MaxStrLength);
+    } else {
+      setPropertiesModalMessage('');
+      setPropsInput(initialPropsInputState);
+      setProperties([...properties, { key, value }]);
+    }
+  };
+
+  const handleDeleteProp = (key: string) => {
+    const found = properties.filter((prop: ItemProperty) => prop.key !== key);
+    setProperties(found);
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLElement>) => {
+    if (e.key === 'Enter') {
+      if (propsInput.key === '' || propsInput.value === '') {
+        setPropertiesModalMessage(propertiesModalMessages.ProvideData);
+      }
+      handleAddMoreProps(propsInput.key, propsInput.value);
+      keyRef?.current?.focus();
+    }
+  };
+
+  return (
+    <>
+      <div className="col-md-6">
+        <div className="input-box">
+          <label htmlFor="property" className="form-label">
+            Properties
+          </label>
+        </div>
+        <div className="d-flex justify-content-between align-items-center newPropertiesInput">
+          <span style={{ color: '#A2A1B2' }}>
+            {properties.length
+              ? `${properties.length} ${properties.length === 1 ? 'property' : 'properties'} added`
+              : 'Add Properties'}
+          </span>
+          <button
+            type="button"
+            onClick={handlePropsModal}
+            className="w-auto"
+            style={{ border: 'none' }}
+          >
+            <i className="feather-plus" style={{ fontSize: '20px' }} />
+          </button>
+        </div>
+      </div>
+      <Modal
+        className="rn-popup-modal placebid-modal-wrapper"
+        show={propsModalOpen}
+        onHide={handlePropsModal}
+        centered
+      >
+        <button type="button" className="btn-close" aria-label="Close" onClick={handlePropsModal}>
+          <i className="feather-x" />
+        </button>
+        <Modal.Header>
+          <h3 className="modal-title fw-light">
+            <b>Add Properties</b>
+          </h3>
+        </Modal.Header>
+        <SpinnerContainer isLoading={isLoading}>
+          <Modal.Body>
+            <p className="text-center">
+              Properties show up underneath your item, are clickable, and can be filtered in your
```

Comment:
This sentence is difficult to understand. Consider to change it to: "The properties that show up underneath your items are clickable and can be filtered..."